packages/flutter_tools/test/resident_runner_test.dart

```diff
@@ -47,7 +47,7 @@ void main() {
 
   setUp(() {
     testRunner = new TestRunner(
-        <FlutterDevice>[new FlutterDevice(new MockDevice())]
+        <FlutterDevice>[new FlutterDevice(new MockDevice(), trackWidgetCreationLocations: false)]
```

Comment:
No tests with `trackWidgetCreationLocations: true` at all?

Reply:
Unfortunately, resident_runner_test doesn't yet work  when the previewDart2 flag is set (configuration issues). When that is fixed, I'll run this test both with and without trackWidgetCreationLocation.